### PR TITLE
Update collapsed broadcast service to save queries for service lists

### PR DESCRIPTION
**Pull request by BjornTwachtmann**

### CODE REVIEW

**Comment by votemike** [comment]
Do we ever want the services back in the order they went in?

**Comment by votemike** [comment]
This will only save a query if ALL the serviceIds requested match ALL of the ones previously requested. Right?

File: src/Service/CollapsedBroadcastsService.php [hunk 1]
```diff
@@ -783,16 +785,57 @@ function ($memo, $broadcast) {
...
+    }
 
-        // Fetch all the used services, keyed by their id
+    private function joinServiceIds(array $serviceIds)
+    {
+        sort($serviceIds, SORT_NUMERIC);
+        return join(",", $serviceIds);
+    }
+
+    private function keyServiceArrayById(array $services)
```
Comment (votemike) [inline src/Service/CollapsedBroadcastsService.php]: Does this return an array of services?

**Comment by BjornTwachtmann** [comment]
Very belated response to the above: Yes, this only saves a query if all the services are the same. However that's an extremely common case (all the services will be the same for anything that's only on BBC one, for example).

**Pull request merged**